$NetBSD$

On NetBSD, do not include librt.  (Why?  All other "UNIX" does, except Mac.)

Do not invoke ldconfig after installation, because it's unportable
(and systems that need it should do it separately, rather than
expecting it to be part of builds).   (From OpenBSD.)

--- lib/CMakeLists.txt.orig	2017-02-20 22:06:57.000000000 +0000
+++ lib/CMakeLists.txt	2017-02-25 12:06:08.924946745 +0000
@@ -48,12 +48,12 @@
 
 set (LIBRARIES ${OPENSSL_LIBRARIES} ${PTHREAD_LIBRARIES})
 
-if (UNIX AND NOT APPLE)
+if (UNIX AND NOT APPLE AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
 	find_library(LIBRT rt)
 	if (LIBRT)
 		set (LIBRARIES ${LIBRARIES} rt)
 	endif (LIBRT)
-endif (UNIX AND NOT APPLE)
+endif (UNIX AND NOT APPLE AND NOT ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
 
 if (WIN32)
 	set (LIBRARIES ${LIBRARIES} ws2_32)
